### PR TITLE
fix(api): patch the country to only accept usa

### PR DESCRIPTION
**Pull request by Orta21**
Ref. metriport/metriport-internal#799

Ref: #799

### Description

Only accept USA as an option for schema

### Release Plan

POINTING TO MASTER

- [ ] Update packages
- [ ] merge

### CODE REVIEW

**Review by leite08 — APPROVED** [review]

File: docs/_snippets/address.mdx [hunk 1]
```diff
@@ -20,7 +20,7 @@
     </ParamField>
 
     <ParamField body="country" type="string" required>
-      Must be "USA".
+      If specified must be "USA", otherwise will default to "USA"
```
Comment (Goncharo) [inline docs/_snippets/address.mdx:23]: nit: punctuation - `will default to "USA".`

File: docs/_snippets/patient-address.mdx [hunk 2]
```diff
@@ -20,7 +20,7 @@
     </ParamField>
 
     <ParamField body="country" type="string">
-      Must be "USA".
+      If specified must be "USA", otherwise will default to "USA"
```
Comment (Goncharo) [inline docs/_snippets/patient-address.mdx:23]: nit: punctuation - `will default to "USA".`

**Review by Goncharo — APPROVED** [review]
2 nits to address - otherwise :shipit:

File: docs/_snippets/address.mdx (hunk 1 above)
Comment (RamilGaripov) [reply docs/_snippets/address.mdx]: nit: punctuation - `If specified, must be "USA"`

File: docs/_snippets/patient-address.mdx (hunk 2 above)
Comment (RamilGaripov) [reply docs/_snippets/patient-address.mdx]: nit: punctuation - `If specified, must be "USA"`

File: docs/_snippets/address.mdx (hunk 1 above)
Comment (Goncharo) [reply docs/_snippets/address.mdx]: lmfao I was thinking this, and was like "aaaahhh fuck it" - good to see we're upholding quality 🚀

**Pull request merged**